crates/xtask/src/tmt.rs

```diff
@@ -387,7 +421,12 @@ pub(crate) fn run_tmt(sh: &Shell, args: &RunTmtArgs) -> Result<()> {
                     distro
                 );
             }
-
+            // Add --filesystem=xfs by default on fedora-coreos
```

Comment:
OK as is but I think we should just put that in the default FCOS container image. As is done in https://gitlab.com/redhat/centos-stream/containers/bootc/-/commit/cfafc9e78a0c9cbdb016113edd6a5dd35d87ab8b
Then we can drop it here.